### ISSUE TITLE
Don't make negative latency a fatal error

### ISSUE THREAD

**Issue by dmgardiner25** [microsoft/dev-tunnels-ssh#70]
I believe there has previously been some discussion between @osortega and @jasongin about this, but we see some occassional RPC disconnects that delays users from being able to connect to their codespaces. Looking into the logs, this is the error that we see:
```
System.ArgumentOutOfRangeException: Measured latency cannot be negative. (Parameter 'latencyMicroseconds')
```
which seems to be coming from [here](https://github.com/microsoft/dev-tunnels-ssh/blob/7623601710eb74327f05dffbdcfa2f7399d4cb56/src/cs/Ssh/Metrics/SessionMetrics.cs#L171-L175). We are still using the old SSH package but are prioritizing the fix here for the coming migration. Would it be possible to make this a non-fatal error or handle it better upstream?

**Comment by jasongin** [comment]
I wish I understood how the latency calculations can come out negative, as it shouldn't be possible. I suspect it may be caused by imprecise time measurement in browser clients; [the MDN doc](performance.now() is coarsened based) says `performance.now()` may be "coarsened" to protect against timing attacks.

Anyway I think it should be fine to downgrade the issue to a log message instead of an exception.

**Comment by jasongin** [comment]
Fixed in version `3.11.20` or later.
